lib/twitter-ads/resources/analytics.rb

```diff
@@ -210,17 +226,17 @@ def fetch_async_job_data(data_url)
       end
 
       def active_entities(account, start_time:, end_time:, **opts)
-        entity_type = name
+        entity            = opts.fetch(:entity, name)
         granularity       = opts.fetch(:granularity, nil)
         start_utc_offset  = opts[:start_utc_offset] || opts[:utc_offset]
         end_utc_offset    = opts[:end_utc_offset] || opts[:utc_offset]
 
-        if entity_type == 'OrganicTweet'
+        if entity == 'OrganicTweet'
           raise "'OrganicTweet' not support with 'active_entities'"
```

Comment:
Nit: Looks like a minor typo. Error message should be `"'OrganicTweet' is not supported with 'active_entities'"`